x/mint/keeper/keeper.go

```diff
@@ -181,7 +179,7 @@ func (k Keeper) AddCollectedFees(ctx sdk.Context, fees sdk.Coins) error {
 	err := k.bankKeeper.SendCoinsFromModuleToModule(ctx, types.ModuleName, k.feeCollectorName, fees)
```

Comment:
Should be renamed to `minted_coins`.

Amount should be `minted_coins` * stakingProportion